recipe/yum_requirements.txt

```diff
@@ -0,0 +1,2 @@
+gtk2-devel
```

Comment:
I don't really like this b/c we end up with a package that links to a lot of system stuff. We need a `gtk` package on `conda-forge`. I am looking into it...